File: test/c++/src/employee.cpp
#include <employee.hpp>

Employee::Employee(const char* name, int id)
	{
	m_name = name;
	m_id = id;
	}
	
Employee::~Employee()
	{
	}
	
std::string Employee::getName()
	{
	return (m_name);
	}
	
int Employee::getId()
	{
	return (m_id);
	}
	

File: test/ant_test/src/java/anttest/Employee.java
package anttest;


public class Employee
	{
	private String m_name;
	private Integer m_id;
	
	
	public Employee(String name, Integer id)
		{
		m_name = name;
		m_id = id;
		}
		
	public String getName()
		{
		return (m_name);
		}
		
	public Integer getId()
		{
		return (m_id);
		}
	}
